### ISSUE TITLE
[BUG] byu-user-info not updating when slotted elements change.

### ISSUE THREAD

**Issue by arasmus8** [byuweb/byu-theme-components#465]
# Description

The byu-user-info component does not change state when slotted elements change. In particular, adding an element into the 'user-name' slot does not change the log in link to the user name.

# Websites Affected

https://dce-dp.sim.byu.edu

# Issue Type

*Is this (add an x in the boxes that apply)*

- [ ] A difference between the components and the [Official Specification](https://brand.byu.edu)?
- [x] A bug, such as a Javascript error, or the UI not rendering properly
    on a page?
- [ ] Inconsistent appearance/behavior between browsers?
- [ ] An issue on mobile browsers?
- [ ] A request for a new feature/enhancement?

# Browsers Affected

*Add an x in all the boxes that apply. Please mark desktop and mobile
browsers separately.*

**We support the last two versions of Chrome, Firefox, Safari, and Edge.**

## Desktop Browsers

- [x] Google Chrome
- [x] Mozilla Firefox
- [ ] Apple Safari
- [ ] Microsoft Edge
- [ ] Other (please specify)

## Mobile Browsers

- [ ] Any browser on iOS
- [ ] Chrome for Android
- [ ] Firefox Mobile for Android
- [ ] Other (please specify)

# Web Site Platform

*What is hosting your website?*

- [ ] Drupal 7
- [ ] Drupal 8
- [ ] Wordpress
- [x ] Custom Site
- [ ] Don't Know

**Comment by blakemorgan** [comment]
What framework and version of the framework are you using? Also, why do you have a `login-url` attribute? We don't support that.

**Comment by ThatJoeMoore** [comment]
Since when do we not support login-url?  Did we drop that in v2?

Joseph Moore
--------------------
Development Operations
Office of Information Technology
*Brigham Young University*
*Office: (801) 422-5436*
*Cell: (801) 850-1749*

**Comment by ThatJoeMoore** [comment]
Yeah, it looks like we're missing all of the slot listeners that were part of v1.  We'll need to resurrect them to handle most SPA use cases.

**Comment by blakemorgan** [comment]
Sounds good. If you have some time on Monday, could you show me a little bit what you mean @ThatJoeMoore? I'm comparing the old code to new code, but am still a little lost.

**Comment by blakemorgan** [comment]
@arasmus8 Go ahead and update your code with the following links and let us know what happens.

```html
<script async src="https://cdn.byu.edu/byu-theme-components/experimental/user-info/byu-theme-components.min.js"></script>
<link rel="stylesheet" href="https://cdn.byu.edu/byu-theme-components/experimental/user-info/byu-theme-components.min.css" media="all">
```

**Comment by ThatJoeMoore** [comment]
Fixed in #466 .

**Comment by arasmus8** [comment]
That's fixed it. Thanks!

**Comment by kebowdog** [comment]
I'm going to re-open this - I've got a few pages where it's not firing the slotchange event when we add an element in the user-name slot.

**Comment by kebowdog** [comment]
For an example see http://mlsapp-new.byu.edu. I've been working on it this morning and can't seem to get the slotchange event to fire.

**Comment by kebowdog** [comment]
Fixed in #470
